mindbender/plugins/collect_project_editorial_info.py

```diff
@@ -0,0 +1,36 @@
+# collect_project_editorial_info.py
+import pyblish.api
+
+
+class CollectMindbenderEditInfo(pyblish.api.ContextPlugin):
+    """Store projects editorial info at the time of publish"""
+
+    label = "Collect Project Edit Info"
+    order = pyblish.api.CollectorOrder
+    hosts = ["maya"]
+    families = [
+        "mindbender.animation"
+    ]
+
+    def process(self, context):
+        import os
+
+        if "environment" not in context.data:
+            context.data["environment"] = dict()
+
+        prefix = "MINDBENDER_"
+        for key, value in os.environ.items():
+            if not key.startswith(prefix):
+                continue
+
+            key = key[len(prefix):]
+
+            # Convert EDIT_IN to EditIn, FPS to Fps.
+            key = "".join(k.title() for k in key.split("_"))
+
+            # Decorate key with "mindbender" to match prev namning convention
+            decoration = prefix.replace("_", "")
```

Comment:
I thought a good ide that since we started calling them "mindbender_{variable name}" inside the bats to continue that practice, but if we dont think that it adds any value then I will just remove the decoration.
and since I didn't know any better cooler way to edit the key before it was assigned then I did the noob way of decoration.
And yea I know there is a function "style" called decorators in python, but I haven't used them yet, I have only read about them.